_posts/2021-02-17-writing_a_cloud-gov_ssp_system_environment.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Writing a cloud.gov SSP: System Environment"
+date: February 17, 2021
+excerpt: We share a sample "System Environment" SSP response for a Low-impact system
+---
+As the cloud.gov compliance lead, I see the occasional support request come through on, "How do I complete the System Environment section of the SSP?"
```

Comment:
Since post author is not displayed, it might make sense to make this more generic - "At cloud.gov, we often get requests from partners going through the ATO process for guidance on how to complete the  System Environment section of the SSP. Since this is a common question, here is an example to get you started. Partners with additional questions can also reach out to support@cloud.gov."

Or something like that.